src/ray/core_worker/core_worker.cc

```diff
@@ -1764,6 +1766,13 @@ void CoreWorker::HandleWaitForRefRemoved(const rpc::WaitForRefRemovedRequest &re
 void CoreWorker::HandleCancelTask(const rpc::CancelTaskRequest &request,
                                   rpc::CancelTaskReply *reply,
                                   rpc::SendReplyCallback send_reply_callback) {
+  // Handle remote cancelation requests.
+  if (!request.remote_object_id().empty()) {
+    RAY_UNUSED(CancelTask(ObjectID::FromBinary(request.remote_object_id()),
+                          request.force_kill()));
+    send_reply_callback(Status::OK(), nullptr, nullptr);
```

Comment:
Why not send the status returned by `CancelTask`?

Reply:
That makes more sense--will add that!